task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskCommentEditor.vue

```diff
@@ -46,14 +50,15 @@
     },
     mounted() {
       this.initCKEditor();
+      const thiss = this;
       $('body').suggester('addProvider', 'task:people', function (query, callback) {
         const _this = this;
-        findUsersToMention(query).then((data) => {
+        findUsersToMention(thiss.task.status.project.id, query).then((data) => {
```

Comment:
What is the expected behaviour when the task is affected to a project ?

Reply:
You mean **not affected** to a project ?

Reply:
Yes when it is not afected to a project